cli/command/cli.go

```diff
@@ -27,26 +27,28 @@ import (
 
 // Streams is an interface which exposes the standard input and output streams
 type Streams interface {
-	In() *InStream
-	Out() *OutStream
+	In() InputStreamer
+	Out() OutputStreamer
 	Err() io.Writer
 }
 
 // Cli represents the docker command line client.
 type Cli interface {
 	Client() client.APIClient
-	Out() *OutStream
+	Out() OutputStreamer
```

Comment:
Could go in the same package tree as the builders, i.e. `cli/internal/test` (to make sure it can't be imported outside of `cli` package 👼 )